apps/tangle-dapp/data/DelegationsPayouts/useDelegations.ts

```diff
@@ -124,7 +133,7 @@ export default function useDelegations(
       isMounted = false;
       sub?.unsubscribe();
     };
-  }, [address]);
+  }, [address, cachedNominations, setCachedNominations]);
```

Comment:
Won't this cause infinite `useEffect` loop, since it will re-trigger on `cachedNominations`, but then we're changing it inside by using `setCachedNominations`? I think we should be using `setCachedNominations((prev) => ...` here instead to avoid any possibility of that.